IDinsight/ask-a-question
# [AAQ-778] Add New dummy data script

Reviewer: @tonyzhao6 
Estimate: 45mins

---

## Ticket

Fixes: [AAQ-778](https://idinsight.atlassian.net/browse/AAQ-778)

## Description

### Goal
The goal of this PR is to add a new script to add. dummy data to AAQ. This data needs to be more realistic with actual questions being asked and more realistic
### Changes
 - Added new script

## How has this been tested?
Tested script on dummy data generated and stored on Google cloud here: https://storage.googleapis.com/ask-a-question-public/generated_questions_updated.csv
To test locally:
- Deploy AAQ using docker-compose (preferably) or using dev setup
- Download and load the Faqs from this google storage csv file: gs://mc_validation_data/faqs_new.csv to deployed AAQ
- Add a few rules to the DB (rules can be arbitrary)
- If using docker-compose deployment: Set POSTGRES_PORT to 5434 (after deployment)
- Use this command to test the script (I recommend updating the script to use a subsample of the df as it can take time): 
`python add_new_data_to_db.py --csv "https://storage.googleapis.com/ask-a-question-public/generated_questions_updated.csv" --domain "localhost/api"` # or "localhost:8000" if using dev setup
- Check dashboard to see if the data has been successfully added

## Checklist

Fill with `x` for completed. 


(Delete any items below that are not relevant)


[AAQ-778]: https://idinsight.atlassian.net/browse/AAQ-778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
